src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,51 @@
+package study;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set 크기 확인 테스트")
+    void size() {
+        int size = numbers.size();
+
+        assertThat(numbers).hasSize(size);
```

Comment:
```suggestion
        assertThat(numbers).hasSize(3);
```
이 테스트의 목적은 단언문의 hasSize와 List의 size가 동일한 결과를 반환하는지가 아닙니다. 
setUp에서 준비한 픽스쳐를 기반으로 의도한 크기가 맞는지 확인하는 것이죠 ㅎ